lib/helpers/client_schema.js

```diff
@@ -295,7 +295,7 @@ export default function getSchema(provider) {
       }
 
       if (responseTypes.has('token') || responseTypes.has('id_token')) {
-        if (!this.grant_types.includes('implicit')) {
+        if (!this.grant_types.includes('implicit') && !configuration.relaxImplicitGrantRequirement) {
```

Comment:
What happens in this case if the client only requests the responseType "token" and not "code"? So there is then no grant type that meets their requirement, but that is no longer checked here. Does it fail at some later point?